docs/api/embed.rst

```diff
@@ -0,0 +1,135 @@
+Embed API
```

Comment:
If we're waiting until embed endpoints are on APIv3, this should go with our other APIv3 endpoint docs at `docs/api/v3.rst`